packages/babel-plugin/src/next.tsx

```diff
@@ -0,0 +1,60 @@
+import * as t from '@babel/types';
+import { declare } from '@babel/helper-plugin-utils';
+import { importSpecifier } from './utils/ast-builders';
+import { visitCssPropPath } from './css-prop';
+import { visitStyledPath } from './styled';
+import { State } from './types';
+
+export default declare<State>((api) => {
+  api.assertVersion(7);
+
+  return {
+    inherits: require('babel-plugin-syntax-jsx'),
+    visitor: {
+      ImportDeclaration(path, state) {
+        if (path.node.source.value === '@compiled/css-in-js') {
+          state.compiledImportFound = true;
+          path.node.specifiers = path.node.specifiers
+            .filter(
+              (specifier) =>
+                specifier.local.name !== 'styled' && specifier.local.name !== 'ClassNames'
```

Comment:
Okay so no matter what import you use (i.e. what flavour of CSS-in-JS you like) we are going to filter out that import and replace it with `CC` and `CS`